views/spaces/index.erb

```diff
@@ -1,5 +1,5 @@
-<% @result.each do |name|%>
-  <%=name%>
+<% @result.each do |space_object|%>
+  <%=space_object.name%><%=space_object.description%>
 <%end%>
 
 <form action='/spaces/new' method='get'>
```

Comment:
need to add a section to the form to include the space description - added to trello board